bin/build-distro

```diff
@@ -18,12 +18,23 @@ shift
 TARGET=${1}
 shift
 
+success=0
+
 cleanup() {
     lxc delete --force "${CNAME}"
+
+    # Only show debug log if build is unsuccessful
+    if [ "${success}" -eq 0 ]; then
+        snap logs -n=1000 lxd
+    fi
 }
 
 trap cleanup EXIT HUP INT TERM
 
+# Set up debug logging
+snap set lxd daemon.debug=true
+snap restart lxd
```

Comment:
`sudo systemctl reload snap.lxd.daemon` would be less impactful, would that help?

Reply:
But `reload` won't restart the LXD daemon with debugging enabled. If at all we should use `restart` or not?

Reply:
sudo systemctl reload snap.lxd.daemon does restart the LXD daemon, but without restarting the instance too.
I always use this way after I enable debugging.

Reply:
OK, I've updated it.